src/rules/max-dependencies.js

```diff
@@ -0,0 +1,49 @@
+import Set from 'es6-set'
+import isStaticRequire from '../core/staticRequire'
+
+const DEFAULT_MAX = 10
```

Comment:
I'm not quite sure howe defaults are handled. I wanted it so that if you enable the rule, but don't specify `{max}` option (is that possible?) it would at least default to a sensible 10.

Not sure if that was the right way to go about it... thoughts?